lib/question.js

```diff
@@ -6,9 +6,9 @@ const jsonStringifySafe = require('json-stringify-safe');
 const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
 const ejs = require('ejs');
 const moment = require('moment');
-const util = require('util');
 const fs = require('fs');
 const unzipper = require('unzipper');
+const fg = require('fast-glob');
```

Comment:
I'm not in love with the fact that we are implicitly supporting all glob syntaxes that `fast-glob` supports. Would could at least turn off all the optional extended syntaxes. That is, set `braceExpansion: false` and `extglob: false`.

Reply:
I can see how `extglob` may potentially be confusing, but I have been using brace expansions occasionally in bash and I don't see how that could be an issue.

Reply:
I'm not worried about confusion, I'm worried about the fact that we are implicitly adding a lot of features that we'll need to support going forward. For example, if we need to switch to a different glob library in the future, we'll need to maintain backwards compatibility with everything we currently support. I'd like that list to be as short as possible 😄.

I'm ok to support brace expansions if you think they are useful -- hopefully they are standard enough that most glob libraries will support them.

Reply:
Done in 2afeeed23. I have set this to a lib-level object to avoid having redundant information in multiple places, so only one place can be changed if these options must be reviewed. I also added an abridged list of allowed patterns in the docs.

Reply:
Excellent, thanks!